### DISCUSSION TITLE
Error when install ipykernel

### DISCUSSION THREAD

**Discussion by yohanespm77** [teamalgoritma/community#703] [Q&A] [answered]
Selamat pagi kak. Saya mengalami error saat menginstall ipykernel di anaconda dengan command pip install ipykernel

<img width="857" alt="image" src="https://github.com/teamalgoritma/community/assets/106010284/e5243549-3604-40d7-ab67-7484a4ac8ee2">

Mohon bantuannya kak, terima kasih 🙏

**Comment by IrfanChairurrachman** [comment]
Dear @yohanespm77 

Terima kasih atas pertanyaannya, akan kami coba telusuri ya Pak permasalahannya.

**Reply by IrfanChairurrachman** [reply]
Dear @yohanespm77 ,

Permasalahan ini dapat disebabkan oleh tidak versi pip yang belum update. Anda dapat upgrade pip dengan command
```
pip install --upgrade pip
```

Mohon pastikan juga Anda update pip dan menginstall ipykernel di dalam environment yang Anda inginkan.

Sekian jawaban dari kami
Semoga membantu

**Reply by yohanespm77** [reply, marked as answer]
Terima kasih atas jawabannya kak @IrfanChairurrachman, sudah bisa kak. Permasalahahnnya ada pada jaringan wifinya ternyata.
